platform/security/idp/security-idp-client/src/main/resources/templates/soap.handlebars

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
We should investigate being able to have exceptions to the license check for templates that might be sent often between systems.